### ISSUE TITLE
when i run the video.py ，i received this error

### ISSUE THREAD

**Issue by milktea-1** [echonet/dynamic#21]
sorry to trouble you ,i wonder that did you run the video.py,i run 45 epochs succuessfully but receive this error
RuntimeError: CUDA out of memory. Tried to allocate 5.38 GiB (GPU 0; 23.70 GiB total capacity; 11.14 GiB already allocated; 2.90 GiB free; 14.29 GiB reserved in total by PyTorch)。
my cmd modelname="r2plus1d_18", frames=32, period=2, pretrained=True, batch_size=8，run_test=True
i run it using NVIDIA RTX3090 (24g) pytorch 1.7 and change the batch_size from 8 to 2  but no use。can you give me some help and what's your environment? Thank you very much

**Comment by milktea-1** [comment]
![报错](https://user-images.githubusercontent.com/40375885/109912147-7ad47b80-7ce6-11eb-90f4-9790142868f5.png)

**Comment by convnet-ai** [comment]
I have the same error. Weird. Was anyone able to run Echonet dynamic all the way through?

**Comment by douyang** [comment]
Unable to reproduce on 3090ti. Should not have significantly different VRAM use by epoch. Make sure to clear cache as needed.

**Comment by 1995Emma** [comment]
> Unable to reproduce on 3090ti. Should not have significantly different VRAM use by epoch. Make sure to clear cache as needed.
I wonder if 2080 is ok. The codes could run on cpu, however when i use the gpu it raised "RuntimeError: CUDA out of memory. Tried to allocate 16.00 MiB (GPU 0; 11.00 GiB total capacity; 9.18 GiB already allocated; 11.94 MiB free; 9.23 GiB reserved in total by PyTorch)" .
I tried different combination of the hyperparameters and it did not work.
Has anyone run into the same problem?
